lib/buffer.js

```diff
@@ -107,7 +111,7 @@ function Buffer(arg, encodingOrOffset, length) {
         'If encoding is specified then the first argument must be a string'
       );
     }
-    return Buffer.allocUnsafe(arg);
+    return allocate(arg);
```

Comment:
@addaleax Not particularly for this change as I measured altogether, but pretty sure that can be kept as-is if needed. May I wonder what's the reasoning behind it?